static/script.js

```
@@ -0,0 +1,38 @@
+
+    let userData = await response.json();
+
+    let userInput = prompt("Enter your name");
+
+    document.getElementById('greeting').innerHTML = `Hello, ${userInput}!`;
+
+    localStorage.setItem('userToken', API_KEY);
+
+    localStorage.setItem('dbPassword', DB_PASSWORD);
+
+    eval(userInput);
```

Biome

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)